controller/tests/controller_server/csi_controller_server_test.py

```diff
@@ -287,17 +309,17 @@ def test_delete_snapshot_with_wrong_secrets(self, storage_agent):
 
         self.request.secrets = {"password": "pass", "management_address": "mg"}
         self.servicer.DeleteSnapshot(self.request, self.context)
-        self.assertEqual(self.context.code, grpc.StatusCode.INVALID_ARGUMENT, "username is missing in secrets")
+        self.assertEqual(self.context.code, grpc.StatusCode.INVALID_ARGUMENT)
         self.assertTrue("secret" in self.context.details)
 
         self.request.secrets = {"username": "user", "management_address": "mg"}
         self.servicer.DeleteSnapshot(self.request, self.context)
-        self.assertEqual(self.context.code, grpc.StatusCode.INVALID_ARGUMENT, "password is missing in secrets")
+        self.assertEqual(self.context.code, grpc.StatusCode.INVALID_ARGUMENT)
         self.assertTrue("secret" in self.context.details)
 
         self.request.secrets = {"username": "user", "password": "pass"}
         self.servicer.DeleteSnapshot(self.request, self.context)
-        self.assertEqual(self.context.code, grpc.StatusCode.INVALID_ARGUMENT, "mgmt address is missing in secrets")
+        self.assertEqual(self.context.code, grpc.StatusCode.INVALID_ARGUMENT)
         self.assertTrue("secret" in self.context.details)

```

Comment:
I think all of this can be simply replaced with a call to `_test_request_with_wrong_secrets`

Reply:
done